src/Main.java

```diff
@@ -13,7 +13,9 @@
 //TODO known issue - sometimes ball gets stuck behind human paddle
 
 public class Main {
+    // code aanpassing
 
+    public int testi = 0;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make testi a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)